src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -146,7 +155,7 @@ const annotatedChlorophyllSamples = computed(() => {
         return {
           ...row,
           color: colorScale(row.value),
-          size: sqrtScale(row.value),
+          size: logScale(row.value),
```

Comment:
Do we need to worry about an undefined for log(0)?

Reply:
It seems to work okay when the chlorophyll value is zero. That is the case in the current data and it just doesn't display anything on that buoy.

Reply:
Sounds good. Maybe add a comment on that line about it, in case it ever cases an issue